test/app/index.html

```diff
@@ -65,31 +65,67 @@
 
 <pre id="results"></pre>
 
+<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=Object.assign"></script>
```

Comment:
Had to add a polyfill for `Object.assign`. Do we care which polyfill we use? We could just copy and paste the polyfill from mdn and include as a separate script tag.